src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -768,7 +769,7 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
 
     /// Lists the identifiers of all known services.
     async fn list_services(&self) -> Result<Vec<String>, anyhow::Error> {
-        let stateful_sets = self.stateful_set_api.list(&self.list_params()).await?;
+        let stateful_sets = self.stateful_set_api.list(&Default::default()).await?;
```

Comment:
Maybe I'm missing some context, but do we not want to only list the ones with the correct `environmentd.materialize.cloud/namespace`?

Reply:
After discussion in chat, this label is not set on the statefulsets, so was previously returning an empty list.